taxonomy-editor-frontend/src/pages/allentries/index.jsx

```diff
@@ -1,20 +1,83 @@
+import { Typography, Button, Box } from "@mui/material";
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import EditIcon from '@mui/icons-material/Edit';
+import { Link } from "react-router-dom";
 import useFetch from "../../components/useFetch";
 import { API_URL } from "../../constants";
-import EntriesList from "./EntriesList";
 
 const Entry = () => {
     const { data: nodes, isPending, isError, isSuccess, errorMessage } = useFetch(API_URL+'nodes');
+    const title = "Test";
     if (isError) {
         return (
             <div className="all-entries">
                 {isError && <div>{errorMessage}</div>}
             </div>
         )
     }
+    if (isPending) {
+        return (
+            <div className="all-entries">
+                {isPending && <div>Loading...</div>}
+            </div>
+        )
+    }
     return (
         <div className="all-entries">
-            {isPending && <div>Loading...</div>}
-            {isSuccess && <EntriesList nodes={nodes} title={"Test"}/>}
+            { isSuccess && 

```

Comment:
nit: one of the benefits of having an status vs different states in the fetch is that it is unsured that these different statuses (and its derived boolean  counterparts `isSuccess`, `isPending`, ...) are mutually exclusive, in this case as you have already checked that it is not loading or is not error then the only remaining option is `isSuccess`, so in principle there shouldn't be a need to check in here.

Reply:
Oh yeah, that makes sense. I'll remove that check.